build/css.js

```diff
@@ -3,7 +3,7 @@ const fs = require('fs');
 
 const postcss = require('postcss');
 
-const postcssConfig = require('../postcss.config');
+const postcssConfig = require('../postcss.config.cjs');
```

Comment:
It looks like postcss.config is using `.js` 🤔

Reply:
`postcss.config.cjs` file in root is common js module. it is different from `.storybook/postcss.config.js`